builds/aks-cleanup/aks-cleanup-nonprod.yml

```diff
@@ -22,6 +30,16 @@ resources:
       name: hmcts/sds-flux-config
       endpoint: 'hmcts'
 jobs:
+  # Job to split labels and set them as job variables for use in folowing steps
+  - job: fetch_pr_labels
+    pool:
+      vmImage: 'ubuntu-latest'
+    steps:
+    - task: Powershell@2
+      inputs:
+        filePath: scripts/pr-label-reader.ps1
```

Comment:
Why add .ps1 and not .sh?

Reply:
@thomast1906  This script contains powershell commands for a PS task too - are we usually naming that with `.sh`?

Reply:
Sorry I mean, did you create the script in PowerShell rather than Bash? Or was it copied from somewhere?

Reply:
Ah - It's a mix of creation and copying some of it from existing [automation runbook](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/1c4f0704-a29e-403d-b719-b90c34ef14c9/resourceGroups/oms-automation/providers/Microsoft.Automation/automationAccounts/CNP-OMS-Automation-NonProd/runbooks/Delete-AKS-Resources/overview)